site/examples/code-highlighting.tsx

```diff
@@ -241,7 +241,7 @@ const SetNodeToDecorations = () => {
 
 const useOnKeydown = (editor: Editor) => {
   const onKeyDown: React.KeyboardEventHandler = useCallback(e => {
-    if (isHotkey('tab', e)) {
+    if (isHotkey('tab', e as any)) {
```

Comment:
Without this a type error happens when building

Reply:
This can be prevented I think by upgrading the `@types` packages for `react` and `react-dom`.